Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -23,21 +23,15 @@ class SnackButton: UIButton {
     let callback: SnackBarCallback?
     fileprivate var bar: SnackBar!
 
-    override open var isHighlighted: Bool {
-        didSet {
-            self.backgroundColor = isHighlighted ? SnackBarUX.highlightColor : .clear
-        }
-    }
-
     init(title: String, accessibilityIdentifier: String, callback: @escaping SnackBarCallback) {
         self.callback = callback
 
         super.init(frame: .zero)
 
         setTitle(title, for: .normal)
         titleLabel?.font = DynamicFontHelper.defaultHelper.DefaultMediumFont
-        setTitleColor(SnackBarUX.highlightText, for: .highlighted)
-        setTitleColor(SettingsUX.tableViewRowTextColor, for: .normal)
+        setTitleColor(.braveOrange, for: .highlighted)
```

Comment:
`highlightText = UIColor.Photon.blue60` seems to be blue not orange, unless we want to update it

Reply:
Decided to update it to reflect Brave highlight/tint colors